docs/release-notes.md

```diff
@@ -0,0 +1,41 @@
+---
+tocdepth: 2
+---
+<!--
+  SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+  SPDX-License-Identifier: Apache-2.0
+-->
+
+# Release Notes
+
+The following sections summarize and highlight the changes for each release.
+For a complete record of changes in a release, refer to the
+[CHANGELOG.md](https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/CHANGELOG.md) in the GitHub repository.
+
+## 0.14.0
+
+### Breaking Changes
+
+- Previously, the model specification in the guardrails configuration file, `config.yml`, supported `model_name` as a top-level field.
+  In this release you must specify the model name in the `model` top-level field or as `model` or `model_name` in
+  the parameters field.
+
+  Before:
+
+  ```yaml
+  models:
+    - type: main
+      engine: nim
+      model_name: meta/llama-3.1-8b-instruct
```

Comment:
Mike please correct me if I'm wrong but I think this was never supported. I'll try to reproduce it using v0.13.0 and let you know.